pub/router.go

```diff
@@ -23,10 +23,10 @@ func ListenAndServe(cfg *Config) error {
 		baseIngressPath,
 	}
 	for _, baseUrl := range baseUrls {
-		router.Any(baseUrl+"/fatman/:fatman/:version/*path", func(c *gin.Context) {
+		router.Any(baseUrl+"/job/:job/:version/*path", func(c *gin.Context) {
 			proxyEndpoint(c, cfg)
 		})
-		router.Any(baseUrl+"/fatman/:fatman/:version", func(c *gin.Context) {
+		router.Any(baseUrl+"/job/:job/:version", func(c *gin.Context) {
```

Comment:
Let's keep both endpoints, new one and the old one for backwards compatibility. This endpoint is probably the most utilized one by the external world.

```
// backwards compatibility endpoints 
router.Any(baseUrl+"/fatman/:job/:version/*path", func(c *gin.Context) {
	proxyEndpoint(c, cfg)
})
router.Any(baseUrl+"/fatman/:job/:version", func(c *gin.Context) {
         proxyEndpoint(c, cfg)
})

router.Any(baseUrl+"/job/:job/:version/*path", func(c *gin.Context) {
	proxyEndpoint(c, cfg)
})
router.Any(baseUrl+"/job/:job/:version", func(c *gin.Context) {
         proxyEndpoint(c, cfg)
})
```